### DISCUSSION TITLE
Creating files from edit mode

### DISCUSSION THREAD

**Discussion by irisdominguez** [gsantner/markor#1853] [Q&A] [answered]
Hi! First of all, thank you so much for all the work that you have done in this software.

I'm currenly using Logseq to keep a personal wiki. On desktop it's ok, but on Android... Quite terrible, to be honest. So I wanted to come back to Markor.

The thing that is keeping me from making the transition is already half implemented in Markor: following a link to create a file. For what I've seen, at least in the common link format it already works from the view mode. Unfortunately, on the edit mode it doesn't, or I don't know how to make it work. There is a "open file" button in the lower bar, but it doesn't do anything for local .md files. 

Is there any way to activate this functionality from the edit mode? Or to keep a live preview open (I'm using mostly a tablet)?

**Comment by gsantner** [comment, marked as answer]
Generally we try best to separate the concerns in Unix fashion - just like Markor is overall a plaintext application, there are certain parts of it for specialized for different things. So the editor is for file editing, the viewer for viewing rendered, the file manager for browsing/creating/deleting/moving files.

While you might have found that special way of how view-mode works, I generally don't see it viable to duplicate the file manager features, (like creating new document) to the editor as well. File management is already insanely complex and I prefer not to mix in another thing and by that also bloating the editor UI.

Though what is of interest for the future is tablet/bigger screen devices improvements, so splitted UI for .showing i.e. filemanager and file editor side by side.
